Vagrantfile

```diff
@@ -55,7 +55,7 @@ end
 
 # TODO(ksweeney): RAM requirements are not empirical and can probably be significantly lowered.
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.hostmanager.enabled = true
+  config.hostmanager.enabled = false
```

Comment:
@granders So, one result of this is that pulling up clusters (i.e. zk or broker, non worker instances) doesn't work properly because hostnames are not defined when the provisioning scripts start the services.

We could just disable hostmanager only for AWS, although that's also kind of weird. We could do it conditionally if you only have workers or something. Thoughts?